views/job-role-list.html

```diff
@@ -25,7 +25,7 @@ <h1 class="mb-4">Job Roles</h1>
                 <tbody>
                     {% for jobRole in jobRoles %}
                     <tr>
-                        <td>{{ jobRole.roleName }}</td>
+                        <td><a href="/job-roles/{{ jobRole.jobRoleId }}" target="_blank" id="{{ jobRole.jobRoleId }}">{{ jobRole.roleName }}</a></td>
```

Comment:
its not a real good idea to have an id as just a number... it should be preceded with with some alphabetic chars.. suggested change is...

id="jobRoleId_{{ jobRole.jobRoleId }}

Reply:
Follow up question - do you need IDs on any of the other columns in this table for UI/accessibility testing ?